test/unit/test_anal_meta.c

```diff
@@ -0,0 +1,615 @@
+
+#include <r_anal.h>
+
+#include "minunit.h"
+
+bool test_meta_set() {
+	RAnal *anal = r_anal_new ();
+
+	r_meta_set (anal, R_META_TYPE_DATA, 0x100, 4, NULL);
+	r_meta_set_string (anal, R_META_TYPE_COMMENT, 0x100, "summer of love");
```

Comment:
I was slightly confused at first, because it may seem that `r_meta_set_string` would set a `R_META_TYPE_STRING`.